report/sql_tasks/tasks/report/create_from_scratch/00_schema/01_create_schemas.jinja2.sql

```diff
@@ -1,3 +1,7 @@
+-- This schema largely exists to dump enums/types into so we can import from
+-- the report schemas in LMS and H without trouble
 DROP SCHEMA IF EXISTS report CASCADE;
-
 CREATE SCHEMA report AUTHORIZATION "{{db_user}}";
+
+DROP SCHEMA IF EXISTS hubspot CASCADE;
+CREATE SCHEMA hubspot AUTHORIZATION "{{db_user}}";
```

Comment:
Should we be dropping the `public` schema here? The `report` schema isn't the main one.

Should we be using `public` at all?

Reply:
It's worth a try but I'd leave it as a separate ticket, we might be relying on some implicit permission on it (and that's probably the best reason to delete it).